app/src/main/java/base/apptrailers/mobile/globant/com/moviesapp/api/MyApi.java

```diff
@@ -0,0 +1,54 @@
+package base.apptrailers.mobile.globant.com.moviesapp.api;
+
+import android.content.ContentValues;
+import android.content.Context;
+
+import base.apptrailers.mobile.globant.com.moviesapp.dataresources.MoviesDataSource;
+import base.apptrailers.mobile.globant.com.moviesapp.dataresources.MyDatabaseHelper;
+import base.apptrailers.mobile.globant.com.moviesapp.entity.Movie;
+
+/**
+ * Created by raul.striglio on 15/09/16.
+ */
+public class MyApi {
+
+    private MoviesDataSource moviesDataSource;
```

Comment:
use interface declaration which provides more flexibility for interchanging implementations